libs/dh/market-participant/feature-gridarea/src/lib/details-header/dh-market-participant-gridarea-details-header.component.html

```diff
@@ -0,0 +1,29 @@
+<!--
+@license
+Copyright 2020 Energinet DataHub A/S
+
+Licensed under the Apache License, Version 2.0 (the "License2");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+
+<ng-container
+  *transloco="let t; read: 'marketParticipant.gridAreas.detailsHeader'"
+>
+  <div>
+    <p class="header-grid-area-code">
+      {{ t("code") + " " + gridArea?.code }}
+    </p>
```

Comment:
suggestion: instead of using a `p` element, consider using a `div`. The `p` is more suitable for blocks of text. In this case it's a simple text value.

Reply:
fixed